python/variable.cpp

```diff
@@ -335,4 +336,33 @@ Mostly equivalent to Variable, see there for details.)");
                    "values, False otherwise.")
           .rtype("bool")
           .c_str());
+
+  m.def(
+      "is_sorted",
+      [](const VariableConstView &x, const Dim dim, const std::string &order) {
+        if (order == "ascending")
+          return is_sorted(x, dim, variable::SortOrder::Ascending);
+        else if (order == "descending")
+          return is_sorted(x, dim, variable::SortOrder::Descending);
+        else
+          throw std::runtime_error(
+              "Sort order must be 'ascending' or 'descending'");
+      },
+      py::arg("x"), py::arg("dim"), py::arg("order") = "ascending",
+      py::call_guard<py::gil_scoped_release>(),
+      Docstring()
```

Comment:
Worth taking the new wrapper approach to docstrings? Could be done as later cleanup as migration process is not completed anyway.

Reply:
Can take this as part of future cleanup. There are also more unrelated places that do not have the new wrappers for docstrings yet.